website/docs/plugins/plugin-typescript-generator.mdx

```diff
@@ -127,3 +127,128 @@ const contract = await Tezos.contract.at<TestContract>(`tz123`);
 const storage = await contract.storage();
 ```
 
```

Comment:
Should this not use the import like the other files?

Reply:
Thanks for pointing that out. I pushed a fix and added some more detail to the PR description